src/overlays/actors/ovl_En_And/z_en_and.c

```diff
@@ -30,22 +29,158 @@ ActorInit En_And_InitVars = {
     (ActorFunc)EnAnd_Draw,
 };
 
-#endif
-
-extern UNK_TYPE D_0600B380;
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_And/func_80C18B90.s")
+static AnimationInfoS sAnimationInfo[8] = {
+    { &gAndStaticAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+    { &gAndIdleAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+    { &gAndWalkAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+    { &gAndRaiseHeadAnim, 1.0f, 0, -1, ANIMMODE_ONCE, 0 },
+    { &gAndRaisedHeadLoopAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+    { &gAndRaiseHandAnim, 1.0f, 0, -1, ANIMMODE_ONCE, 0 },
+    { &gAndRaisedHandLoopAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+    { &gAndRaisedHandWalkAnim, 1.0f, 0, -1, ANIMMODE_ONCE, 0 },
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_And/func_80C18BD8.s")
+static TexturePtr sEyeTextures[4];
+
+s32 EnAnd_ChangeAnimation(EnAnd* this, s32 animIndex) {
```

Comment:
```suggestion
s32 EnAnd_ChangeAnime(EnAnd* this, s32 animIndex) {
```
For these actors animation change functions, we just shorten it to `ChangeAnim`.